pipeline/backend/kubernetes/utils.go

```diff
@@ -23,7 +23,7 @@ import (
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
-	"k8s.io/client-go/tools/clientcmd"
+	client_cmd "k8s.io/client-go/tools/clientcmd" // cspell:disable-line
```

Comment:
Why does this line need to be excluded? Shouldnt it be covered by the regex exclude for import statements?